src/stage1/compiler_utils.jl

```diff
@@ -7,9 +7,7 @@ function Base.push!(cfg::CFG, bb::BasicBlock)
     push!(cfg.index, bb.stmts.start)
 end
 
-if VERSION <= v"1.11.0-DEV.116"
-    Base.getindex(ir::IRCode, ssa::SSAValue) = Core.Compiler.getindex(ir, ssa)
-end
```

Comment:
It just doesn't work on my version of 1.11 without defining this.
idk what was up with 1.11.0DEV.116, maybe a change that was since reverted?